### ISSUE TITLE
Unexpected auto-dial

### ISSUE THREAD

**Issue by banana-dog** [evilcorpltd/aTox#1054]
Long calls are suddenly cut off due to poor internet connection. After that if I open atox from recently used apps, the app automatically starts a call.

**Comment by robinlinden** [comment]
Yeah, connection drops need to be handled during calls, and we should absolutely not automatically try to call again. Will fix. :) Thank you for reporting this!

**Comment by theAkito** [comment]
@banana-dog

We tried to reproduce the issue, but we weren't successful in doing so.

Would you check, if the issue still happens for you?
If yes, could you describe in more detail what exactly is happening and in what way the problem appears?

**Comment by banana-dog** [comment]
@theAkito 
Yes, it still happens sometimes.
Today I tried to reproduce it intentionally: 

1.  call a friend
2. turn off wi-fi
3. wait a minute
4. turn wi-fi on

but the call ends as it should be.

I guess, possible reasons are:

1.  Atox is dead, but the notification is still alive. Opening the notification/opening app from recent used apps starts the call. One can see it - timer in notification starts over. 
2. Maybe atox cannot handle too long calls (around 8 - 14 h) - memory leak? 

I'm an active atox user making calls every day and night. Starting in February it happened 10-20 times.

**Comment by theAkito** [comment]
@banana-dog

Is it possible for you to check this with another device?
Sounds, like this could also depend on what hardware is used.

**Comment by banana-dog** [comment]
@theAkito 
Currently I use realme 8.
Unfortunately I use only one phone and cannot test other hardware.

**Comment by banana-dog** [comment]
There is one more thing, which could be related: sometimes I go offline, but the call does not stop and my status remains "online", however my friends see me "offline".

**Comment by theAkito** [comment]
> sometimes I go offline, but the call does not stop and my status remains "online", however my friends see me "offline".

It takes a while, until status changes are detected. Did you wait at least two minutes & check if it still says online?
